R/BFF_Demux.R

```diff
@@ -8,6 +8,24 @@ utils::globalVariables(
   add = TRUE
 )
 
+#' @title Estimate Multiplet Rate
+#'
+#' @description Estimates multiplet rate by using barcode matrix dimensions and formulae from the Saijita Lab's calculator (https://satijalab.org/costpercell/)
+#' @param barcodeMatrix a barcodes-by-cells matrix of hashing counts
+#' @param num10xRuns The number of lanes the cells are loaded into (expected to be 1)
+
+.EstimateMultipletRate<- function(barcodeMatrix, num10xRuns = 1){
+  numCellsRecovered <- ncol(barcodeMatrix)
+  numCellsLoaded <- 1.75 * numCellsRecovered
+  m <- 4.597701e-06
+  multipletRate <- m*numCellsLoaded/num10xRuns
+  #these seem unncessary for now, but just in case
+  numBatches <- nrow(barcodeMatrix)
```

Comment:
I'm not sure i get what this is trying to do?

Reply:
since we're not using it, at least comment it out and leave a comment as to where it came from

Reply:
Sure thing, these are from the source code of the 10x cost calculator from the Satija lab. Mostly left them in just in case @gboggy wanted to use them for benchmarking. I'll comment them out until we decide they're useful.